SPTDataLoaderTests/SPTDataLoaderServiceTest.m

```diff
@@ -328,21 +328,20 @@ - (void)testAllowingAllCertificates
     NSURLSession *session = [NSURLSession new];
     NSURLSessionTask *task = [NSURLSessionTask new];
     NSURLAuthenticationChallenge *challenge = [NSURLAuthenticationChallenge new];
-    __block NSInteger completions = 0;
+    __block NSInteger nonNilCompletions = 0;
+    void(^NSURLSessionCompletionHandler)(NSURLSessionAuthChallengeDisposition, NSURLCredential *) = ^(NSURLSessionAuthChallengeDisposition disposition, NSURLCredential *credential) {
+        nonNilCompletions += !!credential;
```

Comment:
How about `nonNilCompletions += (credential != nil);`?

I find that a lot easier to read and understand. Up to you.